### PR TITLE
feat : Added subcategory chat bots in aitools and added chatgpt in it…

### LINKED ISSUE #2399: [ADD] Ai Tools/Chat
### Category Name

Ai Tools

### Subcategory Name

Chat

### Additional Context

This subcategory will make it possible to add AI Chats such as ChatGPT, Copilot, Gemini, etc...

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [x] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by devansh-dek**
… #2399

## Changes proposed
Added Chat Bots under ai tools and added link for chat-gpt in it.

## Screenshots 

https://github.com/rupali-codes/LinksHub/assets/83805376/fb59f21f-6bfb-4418-b8ba-1a45cbe03e5e

Closes #2399

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Hi @devansh-dek, could you please add Copilot, Gemini as well?

**Comment by devansh-dek** [comment]
Which copilot should i add
?

**Comment by rupali-codes** [comment]
> Which copilot should i add?

Microsoft copilot

**Comment by devansh-dek** [comment]
After last changes ->
https://github.com/user-attachments/assets/c5299269-00ee-46aa-b0a7-6bad11d1a725

**Review by rupali-codes — COMMENTED** [review]
@devansh-dek could you please rename the file to `chat_bots.json`?

**Comment by devansh-dek** [comment]
> @devansh-dek could you please rename the file to `chat_bots.json`?

Okay i will.

File: database/index.ts [hunk 1]
```diff
@@ -78,6 +78,8 @@ export { default as nlp } from './artificial_intelligence/NLP.json'
 //ai tools
 export { default as design } from './ai_tools/design.json'
 export { default as blog_writing } from './ai_tools/blog_writing.json'
+export { default as chat_bot } from './ai_tools/chat-bot.json'
```
Comment (rupali-codes) [inline database/index.ts:81]:
```suggestion
export { default as chat_bot } from './ai_tools/chat_bots.json'
```

**Comment by rupali-codes** [comment]
@devansh-dek please resolve the merge conflicts

**Comment by rupali-codes** [comment]
@devansh-dek also, change the subcategory name from `chat-bot` to `chat_bots` as per the naming conventions.

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**